src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-field.component.html

```diff
@@ -136,14 +136,9 @@ <h2 mat-dialog-title i18n>Configure Field "{{ entitySchemaField.label }}"</h2>
                   i18n-matTooltip
                 ></fa-icon>
               </mat-label>
-
-              <app-basic-autocomplete
+              <app-entity-type-select
                 formControlName="additional"
-                [options]="typeAdditionalOptions"
-                [optionToString]="objectToLabel"
-                [valueMapper]="objectToValue"
-                [createOption]="createNewAdditionalOptionAsync"
```

Comment:
These variables can also be removed from the component class then, I think?

Reply:
We still use these variables for "Type Details (dropdown options set)".